{"uuid":"1900d3b2-6f3c-461e-ae44-a88b097ad8a2","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["10fb1818-6f07-4083-a301-5fdeaab14d26"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"09ad83bd-a0ea-4a4d-adc1-88aa6b0606d3-attachment","type":"image/png"},{"name":"Myscreenshot","source":"e107146d-3782-4d9e-86e0-6c6f144ace0e-attachment"}],"parameters":[],"start":1696884655536,"stop":1696884655596},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884655596,"stop":1696884655596},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884655597,"stop":1696884655597},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884655598,"stop":1696884655598}],"start":1696884655529,"stop":1696884655599}